src/main/proto/com/kodypay/grpc/pay/v1/pay.proto

```diff
@@ -83,6 +86,7 @@ message RefundResponse {
   string total_amount_refunded = 6;
   string remaining_amount = 7;
   string total_amount_requested = 8;
+  string paymentTransactionId = 9;
```

Comment:
isn't this going to be a refund ID?

Reply:
This is the original transaction id that `kp-cards` has. From how I understand it, `kp-cards` does not create another id for refunds.